싱글 플레이를 누르면 나오는 Dialog를 추가하고 클래스도 추가 했다. 싱글플레이를 누르면 하나의 모달창이 나온다.

soochangoforit opened the pull request:
싱글 플레이를 누르면 나오는 Dialog를 추가하고 클래스도 추가 했다. 싱글플레이를 누르면 하나의 모달창이 나온다.

soochangoforit committed:
싱글 플레이를 누르면 나오는 Dialog를 추가하고 클래스도 추가 했다. 싱글플레이를 누르면 하나의 모달창이 나온다.

Mina-1316 commented:
CPopup 클래스가 싱글 플레이를 클릭했을 때 출력되는 클래스인가요?
만약 맞다면, 클래스 이름을 가독성을 위해 SinglePlayDialog로 변경 해주시면 감사하겠습니다.

리팩토링 사용해서 변경 후, 수정된 파일을 다시 커밋하시면 머지 할게요

soochangoforit committed:
CPopup에서 가독성을 위해 SinglePlayDialog으로 변경했음

soochangoforit commented:
SinglePlayDialog으로 변경 했습니다.

soochangoforit committed:
비행기의 좌표값을 저장할 변수 지정 ,
싱글 플레이 초기화면에 비행기 그림 출력
WASD 방향키 누를때마다 움직인다.
[문제점]
비트맴을 LOAD 하는 방법을 몰라서 우선은
TEXT로 하였다.

soochangoforit commented:
비트맵을 가져오는 방법은 알고 있어서 한번 시도를 해봤는데 , 올바른 ID값을 입력해도 SinglePlayDialog.cpp에 bitmapload부분이 계속 빨간줄 밑줄이 나옵니다.

Mina-1316 commented:
@soochangoforit 일단은 에러 있는 부분은 나중에 구현하기로 하고, 지금은 머지 먼저 할게요

soochangoforit commented:
시도는 해봤는데 에러가 나와서 삭제하고 커밋 했습니다.
